newtmgr/cli/log.go

```diff
@@ -89,16 +106,45 @@ func logShowCmd(cmd *cobra.Command, args []string) {
 			return
 		}
 
-		fmt.Printf("%10s %22s | %11s %11s %6s %s\n",
-			"[index]", "[timestamp]", "[module]", "[level]", "[type]", "[message]")
+		fmt.Printf("%10s %22s | %16s %16s %6s %s\n",
+			"[index]", "[timestamp]", "[module]", "[level]", "[type]",
+			"[message]")
 		for _, entry := range log.Entries {
-			fmt.Printf("%10d %20dus | %11s %11s %6s %s\n",
+			modText := fmt.Sprintf("%s (%d)",
+				nmp.LogModuleToString(int(entry.Module)), entry.Module)
+			levText := fmt.Sprintf("%s (%d)",
+				nmp.LogLevelToString(int(entry.Level)), entry.Level)
+
+			msgText := ""
+			switch entry.Type {
+			case nmp.LOG_ENTRY_TYPE_STRING:
+				msgText = string(entry.Msg)
+			case nmp.LOG_ENTRY_TYPE_CBOR:
+				msgText, err = logCborMsgText(entry.Msg)
+				if err != nil {
+					fmt.Printf("Error decoding CBOR entry: %s; "+
+						"idx=%d",
+						err.Error(), entry.Index)
+					msgText = fmt.Sprintf("%v", entry.Msg)
+				}
+
+			case nmp.LOG_ENTRY_TYPE_BINARY:
+				msgText = fmt.Sprintf("%v", entry.Msg)
```

Comment:
Maybe msgText = hex.EncodeToString(entry.Msg) ?

Reply:
Thanks, Marko.  I have changed the `fmt.Sprintf("%v")` calls to `hex.EncodeToString()`.